doc/releases/changelog-dev.md

```diff
@@ -13,7 +13,21 @@
   be the _remaining_ time evolution complementary to the output for `complementary=False`.
   See the [docstring](https://docs.pennylane.ai/en/stable/code/api/pennylane.pulse.ParametrizedEvolution.html)
   for details.
- 
+
+* The `qml.enable_op_arithmetic` toggle has been introduced to cause dunder methods to return arithmetic
```

Comment:
This is extremely nitpickey, and completely personal preference, but I would prefer `enable_op_math` purely because it's shorter than arithmetic 😅

Reply:
oops just forgot to change the changelog